src/snowflake/connector/ocsp_snowflake.py

```diff
@@ -62,6 +115,20 @@ def __init__(self):
         self.fail_open = False
         self.insecure_mode = False
 
+    def set_event_sub_type(self, event_sub_type):
+        # There can be multiple event_sub_types
+        # in the case where there was a problem
+        # with the cache.
```

Comment:
Maybe a doc string would be better here
```suggestion
        """Sets the event's sub type.
        
        There can be multiple event_sub_types in the case where there
        was a problem, with the cache. Either appends the sub type, or sets it.
        """
```